interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/fragment/FragmentParametersDefinitionExtractor.scala

```diff
@@ -18,18 +18,22 @@ import pl.touk.nussknacker.engine.definition.component.parameter.defaults.{
   DefaultValueDeterminerParameters
 }
 import pl.touk.nussknacker.engine.definition.component.parameter.editor.EditorExtractor
+import pl.touk.nussknacker.engine.definition.component.parameter.validator.{
+  ValidatorExtractorParameters,
+  ValidatorsExtractor
+}
 import pl.touk.nussknacker.engine.graph.expression.Expression
 import pl.touk.nussknacker.engine.graph.node.FragmentInputDefinition.FragmentParameter
 import pl.touk.nussknacker.engine.graph.node.{FragmentInput, FragmentInputDefinition}
 
 /*
- * This class exists as a more lightweight alternative to FragmentParametersCompleteDefinitionExtractor - it doesn't rely on ExpressionCompiler and ValidationContext
- * However, it doesn't validate the parameters' initialValue and valueEditor and doesn't extract (and validate the correctness of) the parameters' validators - use in cases where it's not needed
+ * This class doesn't validate the parameters' initialValue and valueEditor (e.g. values can be of incorrect type).
+ * If they are not validated it is not a breaking issue anyway as a process using these incorrect values will fail validation.
```

Comment:
Is it still the true? IMO we should validate things in the place where user can fix them. What is the reason that we don't validate it?

Reply:
well it would still require validationcontext and expression compiler which was the initial reason behind the `FragmentCompleteDefinitionExtractor` and `FragmentWithoutValidatorsDefinitionExtractor`, maybe i should have left that in the comment